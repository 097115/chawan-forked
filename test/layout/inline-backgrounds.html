<pre style="width: 100%; text-align: center">
<span style="background-color: red">
<span style="text-align: center">testing</span>
test test test <span style="display: inline-block; margin-top: -2em">test</span>
new line

new line
new line
done</span>now another thing
</span>
</pre>
but
<div style="width: 100%; text-align: center">
<br>
<span style="background-color: red">
this works: <span style="text-align: center; padding-left: 2ch">testing</span>
<br>
<br>
<br>
<br>
test test test <span style="display: inline-block">test</span> thing
</div>
final stage
<pre style="width: 100%; text-align: center">
<span style="background-color: red">
<span style="text-align: center">testing</span>
test test test <span style="display: inline-block; margin-top: -2em">test</span>
new line
new line
new line
done</span>now another thing
</span>
</pre>
