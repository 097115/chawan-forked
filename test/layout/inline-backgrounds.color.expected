

                                    [48;2;255;0;0mtesting[49m
                              [48;2;255;0;0mtest test test test[49m
                                    [48;2;255;0;0mnew line[49m

                                    [48;2;255;0;0mnew line[49m
                                    [48;2;255;0;0mnew line[49m
                             [48;2;255;0;0mdone[49mnow another thing


but

                             [48;2;255;0;0mthis works:   testing[49m



                           [48;2;255;0;0mtest test test test thing[49m
final stage


                                    [48;2;255;0;0mtesting[49m
                              [48;2;255;0;0mtest test test test[49m
                                    [48;2;255;0;0mnew line[49m
                                    [48;2;255;0;0mnew line[49m
                                    [48;2;255;0;0mnew line[49m
                             [48;2;255;0;0mdone[49mnow another thing

